### PR TITLE
Yangalex/removeGroup

### PR DESCRIPTION
**Pull request by alexyaang**
This PR addresses issue #3492 by adding the ability to remove an entire container or image group (excluding container compose groups)

### CODE REVIEW

File: ~/.vscode/extensions/theme-dracula
Comment (bwateratmsft) [inline ~/.vscode/extensions/theme-dracula:1]: What's this?

File: package.json [hunk 1]
```diff
@@ -436,6 +436,11 @@
                     "when": "view == dockerImages && viewItem == image",
                     "group": "images_3_destructive@1"
                 },
+                {
+                    "command": "vscode-docker.images.group.remove",
+                    "when": "view == dockerImages && viewItem =~ /imageGroup$/i",
```
Comment (bwateratmsft) [inline package.json:441]: Thinking about it, if the user sees both "Image Group: Remove Image..." and "Image: Remove Image..." in the command palette, confusion is likely to result. I'd suggest a couple things--

1. Don't show this command in the command palette--i.e., in the `menus.commandPalette` part of this file, add one with `"when": "never"` (there are many other examples)
2. Use the same `"category"` property as the other image commands (i.e. `"%vscode-docker.commands.category.dockerImages%"`)
3. Remove the newly-added category string from `package.nls.json`

We did a similar thing when doing commands for compose down--which is a different command if you run it on a file vs. run it on a tree item.

File: src/commands/images/removeImageGroup.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        ext.imagesTree,
+        ImageGroupTreeItem.contextValue,
+        node,
+        nodes
+    );
+
+    const confirmRemove = vscode.l10n.t('Are you sure you want to remove the selected image groups? If there are other tags or child images for images within this group, only the tags will be removed.');
+
+    // no need to check result - cancel will throw a UserCancelledError
+    await context.ui.showWarningMessage (confirmRemove, { modal: true }, { title: 'Remove' });
```
Comment (bwateratmsft) [inline src/commands/images/removeImageGroup.ts:24]:
```suggestion
    await context.ui.showWarningMessage (confirmRemove, { modal: true }, { title: vscode.l10n.t('Remove') });
```

I see the same mistake was already made in `removeImage.ts`, can you fix it there as well?

**Review by bwateratmsft — COMMENTED** [review]
Implementation looks good so far. I think more-or-less the same pattern applies for container groups as well. As discussed, we'll do just containers and images for now, since the need and value is probably far lower for volumes and networks.

File: ~/.vscode/extensions/theme-dracula
Comment (alexyaang) [reply ~/.vscode/extensions/theme-dracula:1]: Great catch! I downloaded a new theme for vscode called Dracula and it somehow showed up here...

File: src/commands/images/removeImageGroup.ts (hunk 2 above)
Comment (alexyaang) [reply src/commands/images/removeImageGroup.ts:24]: Absolutely!

File: package.json [hunk 3]
```diff
@@ -73,6 +73,10 @@
                     "command": "vscode-docker.containers.composeGroup.stop",
                     "when": "config.docker.containers.groupBy == 'Compose Project Name'"
                 },
+                {
```
[the remark this answers is not in the document]
Comment (alexyaang) [reply package.json:76]: liek right above `vscode-docker.debugging.initializeForDebugging` ?

[the remark this answers is not in the document]
Comment (bwateratmsft) [reply package.json:76]: Right above this bit that's added at line 140:
```json
                {
                    "command": "vscode-docker.images.group.remove",
                    "when": "never"
                }
```

[the remark this answers is not in the document]
Comment (alexyaang) [reply package.json:76]: ohh yup that makes sense

**Review by bwateratmsft — APPROVED** [review]

**Pull request merged**